modules/apps/asset/asset-entry-query-processor-custom-user-attributes/src/main/java/com/liferay/asset/entry/query/processor/custom/user/attributes/internal/CustomUserAttributesAssetEntryQueryProcessor.java

```diff
@@ -116,8 +116,8 @@ private void _addUserAttributes(
 						assetCategory.getVocabularyId());
 
 				if (Objects.equals(
-						customUserAttributeName,
-						assetVocabulary.getTitleCurrentValue())) {
+						StringUtil.toLowerCase(customUserAttributeName),
```

Comment:
Why are you doing this

it's works, but, There is a reason why this should be the same?

could you explain me why?

Reply:
After testing, I found that if the user creates a **Custom Field** with a lowercase name like 'region'. The customUserAttributeName's value is 'region'. While the value of assetVocabulary.getTitleCurrentValue() or assetVocabulary.getName() is 'Region'. So the condition will be false. So I did it to cover this case. Please let me know your thought.

Reply:
I'm sorry, the value of **customUserAttributeName** is not **Custom Field** name. It is the value of the **CUSTOM USER ATTRIBUTES** field in Asset Publisher configuration (step 10).